Visual ibac network policies tutorial - new screenshots (#130)
otterize/docs: docs/quick-visual-tutorials/visual-ibac-network-policies.mdx

Look at the access graph again:

![Access graph](/img/quick-tutorials/visual-ibac-network-policies/phase-1.png)

> The thick green line from `frontend` to `recommendationservice`, representing the discovered intent from the network mapper, no longer has a yellow line, but rather a **green**, representing the explicitly declared intent.

Reviewer: ```suggestion
The yellow line from `frontend` to `recommendationservice`, representing the discovered intent from the network mapper, is no longer yellow, but is now green **green**, representing that the connection is allowed.
```
It wasn't green in the first place though, was yellow all along, right?
